src/Support/OMOptions.cpp

```diff
@@ -40,3 +40,8 @@ llvm::cl::opt<int> onnxOpTransformThreshold("onnx-op-transform-threshold",
 llvm::cl::opt<bool> onnxOpTransformReport("onnx-op-transform-report",
     llvm::cl::desc("Report diagnostic info for op transform passes."),
     llvm::cl::init(false), llvm::cl::cat(OMPassOptions));
+
```

Comment:
I would suggest to also have an infrastructure to add options from inside the Accelerator, where maybe we record at a very high level the fact that we have one accelerator, and then in the options, we call each recorded accelerators custom options?